### ISSUE TITLE
Enable option to automate updates, esp. minor ones

### ISSUE THREAD

**Issue by ja601** [dmarmor/epichrome#272]
I'm a huge fan of epichrome but the frequent updates are slowing me down too much. This is because I run half a dozen site specific browsers each on 2 computers. It feels like there's always a new update keeping me from opening the app and since I have to do that for each app, a dozen requests to update is getting too much.

A neat solution would be to give users the option to automatically update, esp. the minor ones. That's common place for many other applications. I'd love to see that here to. I'd give permission for minor updates to be downloaded and put in automatically without user confirmation beyond setting the option. This would making using my epichrome apps much better.

**Comment by dmarmor** [comment]
Hi @ja601, this came up a while back in issue #257. I've changed the whole GitHub-checking system for the upcoming version 2.4.x so that no matter how many apps you have, only one of them will bring up the download dialog. I've also added an app setting whereby apps will automatically update themselves to the latest installed Epichrome without bringing up the update dialog.

Unfortunately, these changes are part of much bigger architectural changes to 2.4.x, so I can't easily add them to the 2.3.x branch, so you'll have to be patient with the frequent dialogs until I can get 2.4.x released. Sorry!

Duplicate of #257
